src/main/kotlin/uk/gov/justice/digital/hmpps/welcometoprison/model/prison/prisonersearch/response/Prisoner.kt

```diff
@@ -0,0 +1,18 @@
+package uk.gov.justice.digital.hmpps.welcometoprison.model.prison.prisonersearch.response
+
+import com.fasterxml.jackson.annotation.JsonFormat
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonInclude.Include.NON_NULL
+import java.time.LocalDate
+
+@JsonInclude(NON_NULL)
+data class Prisoner(
```

Comment:
The prisoner object definition is very similar to the matchPrisonerRersponse object but the latter contains status and isCurrentPrisoner which aren't required by prisoner obj.